### PR TITLE
feat(logging): fix #75 add prometheus in API endpoints

### PR DESCRIPTION
**Pull request by LorenzoS92**
feat(logging): fix #75 add prometheus in API endpoints

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -88,6 +89,11 @@ func Setup() *fiber.App {
 		common.EnvironmentConfig.PasetoKey = middleware.NewRandomPasetoKey()
 	}
 
+	// Setup the prometheus middleware
+	prometheus := fiberprometheus.New(common.EnvironmentConfig.ServiceName)
+	prometheus.RegisterAt(app, "/metrics")
```
[the remark this answers is not in the document]
Comment (LorenzoS92) [reply main.go:93]: Yes, we could share metric publicly in the future but right now, it's better to  hide it via k8s ingress.

File: internal/common/env.go [hunk 2]
```diff
@@ -14,6 +14,7 @@ type Environment struct {
 	CurrentEnvironment string     `env:"ENVIRONMENT" envDefault:"production"`
 	Database           string     `env:"DATABASE_DSN"`
 	PasetoKey          *Base64Key `env:"PASETO_KEY"`
+	ServiceName        string     `env:"SERVICE_NAME" envDefault:"developers_italia_api"`
```
[the remark this answers is not in the document]
Comment (LorenzoS92) [reply internal/common/env.go:17]: Do you have an idea how?

[the remark this answers is not in the document]
Comment (bfabio) [reply internal/common/env.go:17]: Something like `fiberprometheus.New(os.Args[0])`

[the remark this answers is not in the document]
Comment (LorenzoS92) [reply internal/common/env.go:17]: Hmm, we could consider that in the future.
I will keep the PR still open for the issue of /metrics with k8s ingress that of course is important to be solved before merging it

**Comment by bfabio** [comment]
(this actually fixes issue #73)

**Review by bfabio — APPROVED** [review]

**Review by tensor5 — APPROVED** [review]

**Pull request merged**